optashift-employee-rostering-gwtui/src/main/resources/jboss-log4j.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- TODO Remove this file when https://issues.jboss.org/browse/LOGMGR-162 is fixed so logback.xml gets picked up -->
+<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/" debug="false">
```

Comment:
There can only be one logging configuration. Otherwise people don't know which one to use.